reason: Linux/bin/ vendors {listing,fasm} binaries, statically linked for i386

--- SASM.pro
+++ SASM.pro
@@ -31,7 +31,6 @@
     docfiles.path = $$DATADIR/doc/sasm/
 } else {
     binfile.files += sasm
-    binfile.files += Linux/bin/*
     binfile.path = $$BINDIR
     data.files += Linux/share/sasm/*
     data.path = $$DATADIR/sasm/
